src/classes/CW_AudienceGroupRelationParamsCtlrTest.cls

```diff
@@ -0,0 +1,50 @@
+/**
+* File Name   :    CW_AudienceGroupRelationParamsCtlrTest.cls
+* Description :    B-122759 - Process Naming Convention
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 10/07/2014 Amar Reddy     Initial version
+*
+*
+*/
+
+@IsTest
```

Comment:
Style issue: classes and triggers should use the "UpperCamelCase" notation. Suggestion: rename accordingly.